doc/sphinx/changes.rst

```diff
@@ -331,13 +331,11 @@ Commands and options
   (`#15760 <https://github.com/coq/coq/pull/15760>`_,
   by Pierre Roux).
 - **Added:**
-  A deprecation warning that :n:`>` in a field of a :cmd:`Class`
-  and the :g:`Class >` syntax will in the future also
-  declare :ref:`coercions <coercions>` as they do when used in
+  A deprecation warning that the :g:`Class >` syntax, that is currently
+  doing nothing, will in the future
+  declare :ref:`coercions <coercions>` as it does when used in
   :cmd:`Record` commands
   (`#15802 <https://github.com/coq/coq/pull/15802>`_,
-  addresses `#2643 <https://github.com/coq/coq/issues/2643>`_
-  and `#9014 <https://github.com/coq/coq/issues/9014>`_,
```

Comment:
OK, sorry, I keep forgetting that since the switch to rc releases, RMs have ceased to create a specific changelog section for the rc (contrary to how it was done with the beta). Then, your change looks good to me and the only sensible option.